[2025-05-30T01:24:21.391+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T01:24:21.454+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T02:56:00+00:00 [queued]>
[2025-05-30T01:24:21.485+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T02:56:00+00:00 [queued]>
[2025-05-30T01:24:21.485+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-30T01:24:21.536+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): execute_task> on 2025-05-29 02:56:00+00:00
[2025-05-30T01:24:21.568+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1501) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-30T01:24:21.572+0000] {standard_task_runner.py:72} INFO - Started process 1510 to run task
[2025-05-30T01:24:21.561+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flag_check_mysql_dag', 'execute_task', 'scheduled__2025-05-29T02:56:00+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/Sesion3/Ejemplo7.py', '--cfg-path', '/tmp/tmpvaddr__h']
[2025-05-30T01:24:21.576+0000] {standard_task_runner.py:105} INFO - Job 230: Subtask execute_task
[2025-05-30T01:24:21.626+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:24:21.704+0000] {task_command.py:467} INFO - Running <TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T02:56:00+00:00 [running]> on host 31d54648d280
[2025-05-30T01:24:21.706+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:24:21.905+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flag_check_mysql_dag' AIRFLOW_CTX_TASK_ID='execute_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T02:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T02:56:00+00:00'
[2025-05-30T01:24:21.906+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T01:24:21.948+0000] {logging_mixin.py:190} INFO - Flag detectado, ejecutando tarea...
[2025-05-30T01:24:21.949+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-30T01:24:21.960+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T01:24:21.960+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flag_check_mysql_dag, task_id=execute_task, run_id=scheduled__2025-05-29T02:56:00+00:00, execution_date=20250529T025600, start_date=20250530T012421, end_date=20250530T012421
[2025-05-30T01:24:22.032+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-30T01:24:22.051+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
